Update faq for delegated consents (#166)
vippsas/vipps-login-api: vipps-login-from-phone-number-api-howitworks.md

The different steps are described and illustrated with screenshots below.

### 1. Initiate Vipps Login from phone number

The merchant initiates Vipps Login on the users device from their system, based on the user's phone number.
The user receives a push notification from Vipps and clicks the push to open the Vipps app.

If the user has turned off push notifications or for some other reason do not get the push, they can open Vipps themselves and automatically start the login flow.

![User gets push message from Vipps](images/vipps-login-phone-push.png)

### 2. Confirmation

After opening Vipps, the user will be asked to confirm sharing information with the merchant. The first screen will specify the information requested by the merchant.

On this screen, the merchant can choose to show a binding message (confirmation code). This is for added security if the merchant would like their representatives or their customers to have an explicit check that it is the correct transaction that is being accepted.

After confirming the login and sharing of information, the user will be asked for the formal consent to share information from Vipps to the merchant.

> The end user only need to consent once per merchant and the consent applies across all Vipps services. In other words, if the user has already consented to share the

Reviewer: ```suggestion
The end user must only consent once per merchant and the consent applies across all Vipps services. In other words, if the user has already consented to share the
```

requested information with the merchant, no consent will be required, and this step will be skipped.

![User confirm in Vipps](images/vipps-login-confirm.png)

### 3. Delegated consents (optional)

The merchant may choose to include the delegatedConsents scope to collect consents from the end user in Vipps, on behalf of the merchant.
This scope is relevant for merchants who use Vipps Login to enroll end-users into customer club/loyalty program from POS system when the user is present in-store.
If the scope is included in the request, the end user will be asked to give consents according to merchant specifications.

|                       Initial delegated consents screen                       |                           Delegated consents approved                            |
|:-----------------------------------------------------------------------------:|:--------------------------------------------------------------------------------:|
| ![User sees delegated consents in Vipps](images/delegated-consents-step1.png) | ![User confirm delegated consents in Vipps](images/delegated-consents-step2.png) |

> Some of the elements on the delegatedConsents screen can be adjusted to fit the merchant's needs

Reviewer: ```suggestion
Some of the elements on the delegated consents screen can be adjusted to fit the merchant's needs
```

The different steps are described and illustrated with screenshots below.

### 1. Initiate Vipps Login from phone number

The merchant initiates Vipps Login on the users device from their system, based on the user's phone number.
The user receives a push notification from Vipps and clicks the push to open the Vipps app.

If the user has turned off push notifications or for some other reason do not get the push, they can open Vipps themselves and automatically start the login flow.

![User gets push message from Vipps](images/vipps-login-phone-push.png)

### 2. Confirmation

After opening Vipps, the user will be asked to confirm sharing information with the merchant. The first screen will specify the information requested by the merchant.

On this screen, the merchant can choose to show a binding message (confirmation code). This is for added security if the merchant would like their representatives or their customers to have an explicit check that it is the correct transaction that is being accepted.

After confirming the login and sharing of information, the user will be asked for the formal consent to share information from Vipps to the merchant.

> This consent is only needed once per merchant and applies across all Vipps services. In other words, if the user has already consented to share the

Reviewer 2: once per end-user?

Author: Text updates might be better to suggest in confluence, https://confluence.vipps.io/pages/viewpage.action?pageId=3067708835

requested information with the merchant, no consent will be required, and this step will be skipped.

![User confirm in Vipps](images/vipps-login-confirm.png)

### 3. Delegated consents (optional)

The merchant may choose to include the delegatedConsents scope to collect consents from the end user in Vipps, on behalf of the merchant.

> This scope is typically relevant for merchants who employ Vipps Login to enroll users to their loyalty program when the user is present in-store.

Reviewer 2: ```suggestion
This scope is relevant for merchants who use Vipps Login to enroll end-users into customer club/loyalty program from POS system when the user is present in-store.

Some of the elements on the delegatedConsents screen can be adjusted to fit the merchant's needs
 * A top text where the purpose of collecting consents should be stated. This is customizable for you as a merchant.
 * The consents wanted by the merchant (see more [When using delegatedConsents, which consents does Vipps support?](vipps-login-api-faq.md#when-using-delegatedconsents-which-consents-does-vipps-support)).
   These can be individually specified to be mandatory or optional. Mandatory consents must be ticked before the user can complete the login flow. See what consents we currently support, or reach out if you need something else and we'll se what we can do!
 * A bottom text where we provide some contextual information for the end user. The text is fixed, but the links will point to the merchant's terms and conditions, as well as privacy statement.

### 4. Redirect or end the login in Vipps

The merchant controls what happens after the user has confirmed the login and sharing of information.

The merchant can end the login process in Vipps, where the user will receive a confirmation screen that the login is complete.

> Alternatively, the merchant may redirect the user to a webpage in their browser. As the merchant will have an identified user

Reviewer 2: ```suggestion
Alternatively, the merchant may redirect the user to it's wesite. The merchant will then have an identified user. This flow can be used to welcome a user to the customer club, ask for more information or preferences, or show offers or information of relevance.

> Alternatively, the merchant may redirect the user to a webpage in their browser. As the merchant will have an identified user
> entering their web page and as such can control the information given, and ask the user to do further actions to continue the process,
> if relevant. For example, this can be used to welcome a user to the customer club, ask for more information or preferences, or show offers or information of relevance.

Reviewer 2: ```suggestion
```suggestion
Alternatively, the merchant may redirect the user to it's website. The merchant will then have a identified user. This flow can be used to welcome a user to the customer club, ask for more information or preferences, or show offers or information of relevance.
